modules/mssqlserver/build.gradle

```diff
@@ -10,12 +10,12 @@ dependencies {
     compileOnly 'io.r2dbc:r2dbc-mssql:0.8.7.RELEASE'
 
     testImplementation project(':jdbc-test')
-    testImplementation 'com.microsoft.sqlserver:mssql-jdbc:9.5.0.jre8-preview'
+    testImplementation 'com.microsoft.sqlserver:mssql-jdbc:10.2.0.jre8'
 
     testImplementation project(':r2dbc')
     testImplementation 'io.r2dbc:r2dbc-mssql:0.8.7.RELEASE'
 
     // MSSQL's wait strategy requires the JDBC driver
     testImplementation testFixtures(project(':r2dbc'))
-    testImplementation 'com.microsoft.sqlserver:mssql-jdbc:9.5.0.jre8-preview'
+    testImplementation 'com.microsoft.sqlserver:mssql-jdbc:10.2.0.jre8'
```

Comment:
I might be missing something, but why do we have this duplicate line? (we also had it before)
Seems it what introduced in https://github.com/testcontainers/testcontainers-java/pull/2556, maybe just a copy-paste error @rnorth?